assets/stylesheets/host.css

```diff
@@ -9,4 +10,231 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: none;
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.5);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.5s;
+  background-color: white;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 15px;
+  width: 1000px;
+  height: 550px;
+  border: 5px solid;
+  border-radius: 20px;
+}
+
+@keyframes modalFade {
+  0% {
+   opacity: 0%;
+  }
+  100% {
+    opacity: 100%;
+   }
+}
+
+.modal-text {
+  width: 950px;
+  height: 450px;
+  margin: 25px 25px 0px 25px;
+  overflow: scroll;
+}
+
+.modal-activity-title {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  font-size: 48px;
+  font-weight: bold;
+}
+
+.activity-text-full {
+  line-height: 1.5;
+  padding-right: 25px;
 }
+
+.modal-delineation {
+  margin: 15px 0px 15px 0px;
+  border: 1px solid black;
+  opacity: 70%;
+}
+
+.close-modal-btn {
+  background-color: #ed9393;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100px;
+  height: 40px;
+  font-size: 18px;
+  border-radius: 8px;
+}
+
+.close-modal-btn:hover {
+  filter: brightness(0.9);
+}
+
+.close-modal-btn:active {
+  filter: brightness(0.7);
+  box-shadow: 3px 3px rgba(0, 0, 0, 0.8);
+}
+
+.host-header {
+  font-size: 48px;
+  font-weight: bold;
+  margin-bottom: -5px;
+}
+
+.activity-selection {
+  background-color: white;
+  display: flex;
+  flex-direction: row;
+  border: 2px solid;
+  height: 400px;
+  border-radius: 35px;
+  margin-top: 15px;
+  padding: 25px;
+}
+
+.activity-description {
+  width: 40%;
+  font-size: 18px;
+  margin-right: 30px;
+}
+
+.library-model-container {
+  position: relative;
+}
+
+.library-model-container:after {
+  background-color: #aed8f2;
+  content: "\25bc";
```

Comment:
I actually have no idea what this means.  Doesn't mean you need to change it.  Just please explain it to me.

Reply:
I can explain this more on Thursday, but basically, to draw the down arrow on the `<select>` element (at the very right of the `<select>` element), I had to manually add a background color & then place a triangle inside of here. The content - `"\25bc"` - corresponds to drawing the triangle. There *might* be other ways to do this as well, but this was one of the first ways I found that worked - just lmk if you'd like me to change it or something.

Reply:
We talked about this over Slack & I added a quick comment in the CSS.